components/TaskKindSelect.vue

```diff
@@ -50,7 +50,7 @@ function handleChange(selected: (typeof options)[number]) {
     <USelectMenu
       id="task-kind-select"
       :model-value="selected"
-      :options="options"
+      :options="options as unknown as string[]"
```

Comment:
Here the need for the type assertions come from the [`as const`](https://github.com/cytechmobile/radicle-planning-boards/pull/75/files#diff-14fcf142c2c0d64d7e80e9764f8eab4c43f16c02db2bf775381ae09951fa3a06R4) on line 4 of the same file. The `as const` makes sense to keep as is but we could make the type assertion cleaner.

Since the `as const` modifies the array to be `readonly ["All", "Issues", "Patches"]` it no longer extends `string[]` because of the `readonly` part. A cleaner approach would be to define a new helper type `type Writeable<T> = { -readonly [P in keyof T]: T[P] }` (save it as non-exported in the top of this file or under `/types/helpers.ts`.

Here's how it all looks in the end:

![image](https://github.com/cytechmobile/radicle-planning-boards/assets/7542054/10eaea6b-41c4-430d-a23b-11e28138ea17)


The above should also resolve this https://github.com/cytechmobile/radicle-planning-boards/pull/73/files#file-components-taskkindselect-vue-L53 on another PR

![image](https://github.com/cytechmobile/radicle-planning-boards/assets/7542054/1f7166ec-cfe2-4d5d-b57b-388a9acd3c93)